docs/pages/toolpad/core/api/sign-out-button.json

```diff
@@ -1,5 +1,5 @@
 {
-  "props": {},
+  "props": { "children": { "type": { "name": "node" } } },
```

Comment:
Proptypes are now being generated for the components which don't match the name of the folder (secondary ones), I think that should be a good thing?